x/warp/config.go

```diff
@@ -0,0 +1,213 @@
+// (c) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package warp
```

Comment:
This is meant to be moved directly into the precompile package as the contract implementation.

I think we should either have one package with everything warp related as a sub-package or separate it into the warp contract and everything else as it is now. I prefer this approach because it follows the convention of having each precompile contract implementation within `precompile/contracts/XXX`

Reply:
I agree it's better to have this under precompile package.